cmd/homerunner/test/test.mjs

```diff
@@ -0,0 +1,36 @@
+import { Homerunner } from "homerunner-client";
+
+const run = async () => {
+    const client = new Homerunner.Client();
+    console.log("homerunner base url:", client.baseUrl);
+    console.log("creating homeserver...");
+    const blueprint = await client.create({
+        base_image_uri: "complement-dendrite",
+        blueprint_name: "one_to_one_room",
+    });
+    console.log(blueprint);
```

Comment:
I presume this is intentionally left in, if it is can we have some context before it like:

```suggestion
    console.log("Client.create responded with", blueprint);
```